contrib/libskk/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "libskk"
+pkgver = "1.0.5"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr"]
```

Comment:
this is gnu_configure, and for it you won't need --prefix

it's quite old so it might be hard to reconf, in that case configure_gen = []. but it looks old enough to need a reconf to work correctly with new clang anyway..